Hmm can' t seem to run tests. Not rly sure why though. It works without your changes though
```
$ make test
no vagd instance is running
STARTING TEST
Traceback (most recent call last):
  File "/tmp/vagd/test/./test.py", line 5, in <module>
    import vagd.virts.pwngd
  File "/tmp/vagd/test/vagd/__init__.py", line 2, in <module>
    from vagd.virts import *
  File "/tmp/vagd/test/vagd/virts/__init__.py", line 1, in <module>
    from vagd.virts.dogd import Dogd
  File "/tmp/vagd/test/vagd/virts/dogd.py", line 6, in <module>
    from vagd.virts.pwngd import Pwngd
  File "/tmp/vagd/test/vagd/virts/pwngd.py", line 11, in <module>
    from vagd import helper
  File "/tmp/vagd/test/vagd/helper.py", line 6, in <module>
    from vagd.virts.pwngd import Pwngd
ImportError: cannot import name 'Pwngd' from partially initialized module 'vagd.virts.pwngd' (most likely due to a circular import) (/tmp/vagd/test/vagd/virts/pwngd.py)
test unsuccessful, pls fix
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "vagd"
-version = "1.6.3"
+version = "1.7.0"
 authors = [{ name = "0x6fe1be2" }]
 description = "VirtuAlization GDb integrations in pwntools"
 readme = "README.md"

--- a/src/vagd/virts/__init__.py
+++ b/src/vagd/virts/__init__.py
@@ -1,3 +1,4 @@
+from vagd.virts.cogd import Cogd
 from vagd.virts.dogd import Dogd
 from vagd.virts.pogd import Pogd
 from vagd.virts.logd import Logd

--- a/test/test.py
+++ b/test/test.py
@@ -200,7 +200,7 @@ def virts():
 
     os.system("vagd clean")
 
-  if not args.NOPOGD:
+  if args.PODMAN:
     if os.path.exists(Pogd.LOCKFILE):
       os.remove(Pogd.LOCKFILE)
     stage("Testing Podman for Ubuntu")

